build/common.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <!-- Packages can have independent versions and only increment when released -->
-    <Version>3.0.34$(VersionSuffix)</Version>
+    <Version>3.0.35$(VersionSuffix)</Version>
```

Comment:
We haven't released official version 3.0.34 so we're still in the same minor version train and shouldn't need to update